csuntechlab/iSTART
# STR-180 -- Add back routeguard & Restrict development tools

# Description
Added back the vue router navigation guard to secure front-end routes & Restricted development tools to dev environments or demo mode.
  
# Testing
1. Ensure you are only able to access the routes you are able to as `admin` or `user` credentials.
2. Ensure the development tools to skip slides is not enabled unless you are an `admin` in **demo mode**.
